packages/yourloops/components/chat/chat-widget.tsx

```diff
@@ -0,0 +1,231 @@
+    setShowPicker(false);
+    setInputText(inputText + emojiObject.emoji);
+  };
+
+  const resetInputSize = () => {
+    if (!content.current || !footer.current || !input.current) {
+      throw new Error("Cannot find elements for resize");
+    }
+    content.current.style.height = "420px";
+    footer.current.style.height = "40px";
+    input.current.style.overflow = "hidden";
+    input.current.style.height = "24px";
+  };
+
+  const sendMessage = async () => {
+    const session = authHook.session();
+    if (session !== null) {
+      await sendChatMessage(session, teamId, patientId, inputText);
+      const messages = await getChatMessages(session, teamId, patientId);
+      setMessages(messages);
+      setInputText("");
+      resetInputSize();
+      if (input.current) {
+        input.current.value = "";
+      }
+    }
+  };
+
+  const inputHandler = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setInputText(event.target.value);
+    event.target.style.height = "1px";
+    event.target.style.height = (event.target.scrollHeight) + "px";
+    if (!content.current || !footer.current) {
+      throw new Error("Cannot find elements for resize");
+    }
+    //values for scrollHeight are 24 one line, 44 2 lines and 64 3 lines
+    //we display the scrollbar only when 3 lines are present
+    if (event.target.scrollHeight > 44) {
+      event.target.style.overflow = "auto";
+      content.current.style.height = "380px";
+      footer.current.style.height = "80px";
+    } else if (event.target.scrollHeight > 24) {
+      content.current.style.height = "400px";
+      footer.current.style.height = "60px";
+      event.target.style.overflow = "hidden";
+    } else {
+      resetInputSize();
+    }
+  };
+
+  return (
+    <Card className={`${classes.chatWidget}`} >
+      <div className={`${classes.chatWidgetHeader}`}>
+        <EmailOutlinedIcon className={`${classes.icon}`}/>
+        <span className={`${classes.chatWidgetHeaderText}`}>Messages (+1)</span>
+      </div>
+      <div ref={content} id={"chatWidgetContent"} className={`${classes.chatWidgetContent}`}>
+        {messages.map(
+          (msg): JSX.Element => (
+            <ChatMessage key={Math.random()} text={msg.text} isMine={msg.author === userId || role === "hcp" && msg.author !== patientId }/>
+          ))
+        }
+      </div>
+      <div>
+        { showPicker &&
+          <Picker pickerStyle={{ width: "100%" }} onEmojiClick={onEmojiClick} />
+        }
+      </div>
+      <div ref={footer} id={"chatWidgetFooter"} className={`${classes.chatWidgetFooter}`}>
+        <button className={`${classes.iconButton}`} onClick={() => setShowPicker(true)}>
+          <SentimentSatisfiedOutlinedIcon/>
+        </button>
+        <textarea ref={input} value={inputText} rows={1} id={"chatWidgetInput"} className={`${classes.chatWidgetInput}`}
+          onInput={inputHandler} placeholder={"Commencer à écrire ..."}/>
```

Comment:
Placeholder should be handle by translation

Reply:
I'm going to pass translation for every string